trino/client.py

```diff
@@ -869,82 +869,79 @@ def map(self, value) -> Optional[float]:
         return float(value)
 
 
-class TemporalValueMapper():
-    def _get_number_of_digits(self, column):
-        args = column['arguments']
-        if len(args) == 0:
-            return 3, 0
-        ms_size = args[0]['value']
-        if ms_size == 0:
-            return -1, 0
-        ms_to_trim = ms_size - min(ms_size, 6)
-        return ms_size, ms_to_trim
-
-
-class TimeValueMapper(ValueMapper[time], TemporalValueMapper):
-    def __init__(self, column):
-        pattern = "%H:%M:%S"
-        ms_size, ms_to_trim = self._get_number_of_digits(column)
-        if ms_size > 0:
-            pattern += ".%f"
-        self.pattern = pattern
-        self.time_size = 9 + ms_size - ms_to_trim
+def round_millis(value: str, precision: int) -> str:
+    if len(value) > precision:
```

Comment:
This makes assumption that input is a non-fractional number? Let's defensively verify our assumption. Nothing prevents the caller from using the function in other contexts in future.